_schemaV2.graphql

```diff
@@ -6826,6 +6836,18 @@ type MyCollectionConnection {
   totalCount: Int
 }
 
+input MyCollectionCreateArtworkInput {
+  artistIds: [String]!
+  clientMutationId: String
+  dimensions: String!
+  medium: String!
+}
+
+type MyCollectionCreateArtworkPayload {
+  artwork: Artwork
+  clientMutationId: String
```

Comment:
**comment [I'm learning]:**

For anyone like me wondering what `clientMutationId` is about, [the docs](https://relay.dev/docs/en/classic/classic-api-reference-relay-graphql-mutation#constructor) say this:

> That input argument should contain a (string) "clientMutationId" property for the purposes of reconciling requests and responses (automatically added by the Relay.GraphQLMutation API).

_Comment format based on [conventionalcomments.org](https://conventionalcomments.org/)_